lib/set_member_subject_selector.rb

```diff
@@ -11,7 +11,7 @@ def initialize(workflow, user)
 
   def set_member_subjects
     to_classify = select_set_member_subjects_to_classify
-    unless to_classify.exists?
+    if !to_classify.exists? && user
```

Comment:
yeah - sorry. that makes sense. now i look at it i think this is wrong anyway. i should not be queueing possibly retired data otherwise we'll overclassify. but instead just returning an empty scope that will trigger a db select anyway.